requirements-tests.txt

```diff
@@ -0,0 +1,4 @@
+freezegun
```

Comment:
overall, I'd totally set the versions for this, not doing so may be prone to errors when running tests. In doubt, check which ones are you currently using and add those.

Reply:
I already pinpointed all the requirements from this file